pkg/operator/starter.go

```diff
@@ -180,21 +253,20 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 	clusterOperatorStatus := status.NewClusterOperatorStatusController(
 		targetName,
 		[]configv1.ObjectReference{
-			{Resource: "namespaces", Name: targetNamespace},
-			{Resource: "namespaces", Name: operatorNamespace},
+			{Resource: "namespaces", Name: defaultOperandNamespace},
```

Comment:
If `controlPlaneNamespace != defaultOperandNamespace` we should have `controlPlaneNamespace` here.

Reply:
I don't think so. This is a list of objects related to CSI snapshot ClusterOperator in the guest cluster, `controlPlaneNamespace` is not in the guest cluster.